internal/pkg/testutil/testing.go

```diff
@@ -9,26 +9,31 @@ import (
 	"github.com/segmentio/terraform-docs/internal/pkg/tfconf"
 )
 
-// GetExpected returns 'example' Module and expected Golden file content
-func GetExpected(goldenFile string) (*tfconf.Module, string, error) {
-	path, err := getExampleFolder()
+// GetModule returns what is generated by the test params
+func GetModule(options *tfconf.Options) (*tfconf.Module, error) {
+	path, err := GetExampleFolder()
```

Comment:
Although I didn't get why you made `GetExampleFolder` exported since it's only being used in `internal/pkg/testutil/testing.go`. I'd suggest to revert it back to `getExampleFolder` (effectively make it private)

Reply:
Oops leftover from something else I had tried. Thanks for catching that.